code/datums/tutorial/xenomorph/xenomorph_basic.dm

```diff
@@ -0,0 +1,229 @@
+	else if(xeno.health > WAITING_HEALTH_THRESHOLD)
+		reach_health_threshold()
+		UnregisterSignal(xeno, COMSIG_XENO_START_EMIT_PHEROMONES)
+	else
+		UnregisterSignal(xeno, COMSIG_XENO_START_EMIT_PHEROMONES)
+		message_to_player("Well done. Recovery Pheromones will significantly speed up your health regeneration. Rest or wait until your health is at least [WAITING_HEALTH_THRESHOLD].")
+		message_to_player("Pheromones also provide their effects to other xenomorph sisters nearby!")
+		RegisterSignal(xeno, COMSIG_XENO_ON_HEAL_WOUNDS, PROC_REF(reach_health_threshold))
+
+/datum/tutorial/xenomorph/basic/proc/reach_health_threshold()
+	SIGNAL_HANDLER
+	if(xeno.health < WAITING_HEALTH_THRESHOLD)
+		return
+
+	UnregisterSignal(xeno, COMSIG_XENO_ON_HEAL_WOUNDS)
+
+	message_to_player("Good. Well done.")
+	message_to_player("A hostile human or \"tallhost\" has appeared. Use your <b>harm intent</b> to kill it in melee!")
+	update_objective("Kill the human!")
+
+	var/mob/living/carbon/human/human_dummy = new(loc_from_corner(7,7))
+	add_to_tracking_atoms(human_dummy)
+	add_highlight(human_dummy, COLOR_RED)
+	RegisterSignal(human_dummy, COMSIG_MOB_DEATH, PROC_REF(on_human_death_phase_one))
+
+/datum/tutorial/xenomorph/basic/proc/on_human_death_phase_one()
+	SIGNAL_HANDLER
+
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+
+	UnregisterSignal(human_dummy, COMSIG_MOB_DEATH)
+	message_to_player("Well done. Killing humans is one of many ways to help the hive.")
+	message_to_player("Another way is to <b>capture</b> them. This will grow a new xenomorph inside them  which will eventually burst into a new playable xenomorph!")
+	addtimer(CALLBACK(human_dummy, TYPE_PROC_REF(/mob/living, rejuvenate)), 8 SECONDS)
+	addtimer(CALLBACK(src, PROC_REF(proceed_to_tackle_phase)), 10 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/proceed_to_tackle_phase()
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+	remove_highlight(human_dummy)
+	RegisterSignal(human_dummy, COMSIG_MOB_TAKE_DAMAGE, PROC_REF(on_tackle_phase_human_damage))
+	RegisterSignal(human_dummy, COMSIG_MOB_TACKLED_DOWN, PROC_REF(proceed_to_cap_phase))
+	message_to_player("Tackle the human to the ground using your <b>disarm intent</b>. This can take up to four tries as a drone.")
+	update_objective("Tackle the human to the ground!")
+
+/datum/tutorial/xenomorph/basic/proc/on_tackle_phase_human_damage(source, damagedata)
+	SIGNAL_HANDLER
+	if(damagedata["damage"] <= 0)
+		return
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+	if(human_dummy.health < 100)
+		message_to_player("Don't harm the human!")
+		human_dummy.rejuvenate()
+
+/datum/tutorial/xenomorph/basic/proc/proceed_to_cap_phase()
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+
+	UnregisterSignal(human_dummy, COMSIG_MOB_TACKLED_DOWN)
+
+	human_dummy.Stun(9 HOURS)
+	human_dummy.KnockDown(9 HOURS)
```

Comment:
Do we have something to apply an indefinite stun & knockdown?

Reply:
```
human_dummy.apply_status_effect(/datum/status_effect/incapacitating/stun)
human_dummy.apply_status_effect(/datum/status_effect/incapacitating/knockdown)
```